src/main/java/com/cloudbees/jenkins/support/impl/AboutJenkins.java

```diff
@@ -31,8 +31,7 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.PrintWriter;
-import java.lang.management.ManagementFactory;
-import java.lang.management.RuntimeMXBean;
+import java.lang.management.*;
```

Comment:
try to avoid ```.*``` imports as that could make future code reviews harder to do when you can't easily find what package a class comes from.